empyrical/utils.py

```diff
@@ -458,15 +498,15 @@ def default_returns_func(symbol, start=None, end=None):
     if symbol == 'SPY':
         filepath = data_path('spy.csv')
         rets = get_returns_cached(filepath,
-                                  get_symbol_returns_from_yahoo,
+                                  get_symbol_returns_from_quandl,
                                   end,
                                   symbol='SPY',
-                                  start='1/1/1970',
+                                  start=start,
                                   end=datetime.now())
-        rets = rets[start:end]
+        rets = rets[rets.index.isin(pd.bdate_range(start, end))]
     else:
-        rets = get_symbol_returns_from_yahoo(symbol, start=start, end=end)
-
+        rets = get_symbol_returns_from_quandl(symbol, start=start, end=end)
+    rets.sort_index(inplace=True)
```

Comment:
Does quandl not give a sorted index?

Reply:
No. Curiously, it is returned in _reverse_ chronological order.

![screen shot 2018-05-17 at 2 44 17 pm](https://user-images.githubusercontent.com/19851673/40197251-e28c7722-59e0-11e8-9989-5c20cbec105c.png)